src/main/java/uk/gov/hmcts/reform/civil/service/claimstore/ClaimStoreService.java

```diff
@@ -42,6 +43,15 @@ public List<DashboardClaimInfo> getClaimsForDefendant(String authorisation, Stri
         }
     }
 
+    public DefendantLinkStatus isOcmcDefendantLinked(String caseReference) {
+        try {
+            return claimStoreApi.isDefendantLinked(caseReference);
+        } catch (Exception e) {
+            log.error(e.getMessage());
```

Comment:
maybe pass e to log.error to keep the original exception

Reply:
Ah, will add e